index.html

```diff
@@ -2078,40 +2079,37 @@ <h2>Authentication</h2>
 }
         </pre>
 
-        <div class="note" title="Uses of authentication">
-          <p>
+        <p>
 If authentication is established, it is up to the <a>DID method</a> or other
 application to decide what to do with that information. A particular <a>DID
 method</a> could decide that authenticating as a <a>DID controller</a> is
 sufficient to, for example, update or delete the <a>DID document</a>. Another
```

Comment:
I just noticed this point here. It's likely useful to move this to the top of the verification methods section because it remains true for all verification methods, not only for the authentication VM.

Reply:
Hmm, I wouldn't say that it is true for all DID Methods. Authentication is about *authentication*... not, for example, authorization... which is what the capabilityInvocation and capabilityDelegation fields are about. Perhaps I'm not understanding your point, @kdenhartog?